packages/@webex/widget-cc-station-login/src/hooks.ts

```diff
@@ -0,0 +1,3 @@
+export const useStationLogin = () => {
+  return {name: 'StationLogin'};
```

Comment:
can we confirm the naming of this file as hooks if its going to be just a helper file for components should be name it utils or something as it gets confused with react hooks

if we going to import react stuff here i assume its ok

Reply:
Changed to helper.ts